db-pull-on-boot/templates/db-pull.sh.j2

```diff
@@ -16,6 +16,14 @@ trap cleanup EXIT
 
 touch $LOCK_FILE
 
+echo "Downloading s3 database..."
+aws --no-progress s3 cp "s3://{{ blockchain_backup_bucket }}/{{ network }}/blockchain_{{ db_version }}_{{ network }}.sqlite" "{{ chia_root }}/blockchain_{{ db_version }}_{{ network }}.sqlite"
+aws --no-progress s3 cp "s3://{{ blockchain_backup_bucket }}/{{ network }}/height-to-hash" "{{ chia_root }}/height-to-hash"
+aws --no-progress s3 cp "s3://{{ blockchain_backup_bucket }}/{{ network }}/sub-epoch-summaries" "{{ chia_root }}/sub-epoch-summaries"
+sudo chown ubuntu:ubuntu "{{ chia_root }}/blockchain_{{ db_version }}_{{ network }}.sqlite"
```

Comment:
for ubuntu:ubuntu - I think we actually have a `user` and `group` ansible var that we should use instead of assuming the user

Reply:
In chia-blockchain, the defaults are here: https://github.com/Chia-Network/ansible-roles/blob/main/chia-blockchain/defaults/main.yml#L6

We should use the same variable names in this role, and add the same ubuntu defaults, so that anyone using a combination of these roles can set these easily